enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/snapshot/PersistentSnapshotDownloader.java

```diff
@@ -86,6 +86,7 @@ public void run()
         }
         catch ( InterruptedException e )
         {
+            Thread.interrupted();
```

Comment:
I think this line will to nothing. It should be `Thread.currentThread().interrupt()` if intention is to restore the interruption status.